src/components/MySpace/MySpace.test.tsx

```diff
@@ -246,20 +251,11 @@ describe('My Space Component', () => {
 
     userEvent.click(buttons[0])
 
-    // Wrapping this in act due to https://github.com/apollographql/apollo-client/issues/5920
     await act(async () => {
       jest.runAllTimers()
     })
 
-    jest.useRealTimers()
```

Comment:
I _think_ this should stay in?

Reply:
i added it to `afterEach(() => ...` at the beginning of the file, so we wouldn't have to repeat in any other tests, but I can keep it here if you'd like!

Reply:
ohh ok, in that case never mind!